### ISSUE TITLE
The output of the model has no gradients

### ISSUE THREAD

**Issue by wzjin2017** [ayooshkathuria/pytorch-yolo-v3#112]
Hi! Thank you for building up this pytorch project. I am modifying the project to my own needs, which requires the output of the model has gradients. However, the darknet output's requires_grad is False. Is it possible to have it to be True?
